### ISSUE TITLE
Ship Set stay the same for new game

### ISSUE THREAD

**Issue by frank-zago** [BrokenRegistry/Rotp-Fusion#113]
Start a "New Game", and select a different race. The Ship Set stays the same, but Homeworld and Leader will change with the new selected species.

**Comment by frank-zago** [comment]
The "Original" (also called "Displayed Race") setting will select the current race, but this is all confusing. Why does one stay but not the other two? I'd remove the "displayed race" option and put the species name instead.

**Comment by BrokenRegistry** [comment]
By default, each species has a dedicated ship set.
While they are all well done, players may like some more than others.
That is why I added this option to allow the player to select their preferred ship set.
Selecting "Original" means that when the species is changed, the ship set will follow. (This is probably not the best name, but one has to be chosen)
If you wish to select a different set, they will be referenced by their original dedicated species name.
Navigating thru the species will then not change the selected chip set.
The selection in this panel has priority over the custom species, because it's the last you will see.
If you want the custom species ship set to apply, you will have to select "Original". If the Custom species is linked to a ship set, it'll the be displayed as "CR:ship set". Choosing "displayed race" is the same as "I don't care", and the "Original" selection will be the one of the species selected to be displayed.
Note:
An empire is tied to two species: one for the animations displayed, and one for their abilities. By default, they are the same. The separation was made by Ray for his "Random AI" option, where you don't know who you are fighting.
I used this feature to create the custom species that are tied to abilities. That's why a "Display" species must be chosen.
The icon panel ship set is tied to the "Display" species, while the custom species ship set is tied to the "Abilities" species.
The game ship set attribution look first at the "Display" species.
- If the payer has selected a set, it will be used.
- If it's "Original", then the "ability" ship set will used.
- If it's "Displayed race", then the default "display" ship set will be used.

Yes, it's a bit confusing, but hiding "Original" behind the final result could be even more confusing!

I can also add a "Prefered Display species", but the 50 empires can have the same "abilities" species while "Display" species are limited to 5.
